src/utils/oauth/GithubProvider.ts

```diff
@@ -107,12 +107,12 @@ const GithubProvider: IProvider<boolean> = {
     /* POST request to get the access token from GitHub */
     await fetch(AuthorizeUrl, {
       headers: {
-        "Accept": "application/json",
+        Accept: "application/json",
         "Access-Allow-Credentials": "True",
         "Access-Control-Allow-Methods": "POST",
         "Access-Control-Allow-Origin": "*",
         "Content-Type": "application/json",
-        "Vary": "Origin",
+        Vary: "Origin",
```

Comment:
Why has this change been reverted?

Reply:
In case we have this problem in the future again, add "// prettier-ignore" before the code part, to disable prettier.